.github/workflows/code-scanning.yml

```diff
@@ -0,0 +1,39 @@
+name: Code Scanning
```

Comment:
what's that?

Reply:
a cron workflow that uses codeql to scan the repo for any vulnerabilities.

regarding prettier, check package.json its configured over there in `lint-staged` section